bc_obps/registration/tests/integration/test_operation_registration.py

```diff
@@ -0,0 +1,273 @@
+from datetime import timedelta
+
+import pytest
+from django.db.models import Q
+from model_bakery import baker
+from registration.models import RegistrationPurpose, FacilityDesignatedOperationTimeline, Operation
+from registration.tests.constants import MOCK_DATA_URL
+from registration.tests.utils.bakers import bc_obps_regulated_operation_baker
+from registration.tests.utils.helpers import CommonTestSetup, TestUtils
+from registration.utils import custom_reverse_lazy
+
+
+class TestOperationRegistration(CommonTestSetup):
+    def setup_method(self):
+        super().setup_method()
+        self.approved_user_operator = baker.make_recipe('utils.approved_user_operator', user=self.user)
+        self.bcghg_id = baker.make_recipe('utils.bcghg_id')
+        self.swrs_facility_id = 1003
```

Comment:
Are we still using swrs_facility_id in reg2 now that we have a Facility model? I don't remember seeing it on any of the forms

Reply:
Good point!